tetrateci/1.13/test/skip.d/eks

```diff
@@ -22,9 +22,9 @@
 #
 # indicates that tests for the package `<pkg>` should not be run at all.
 
-istio.io/istio/tests/integration/helm/upgrade=TestRevisionTagsUpgradeFromPrevious
-MinorRelease
-
+istio.io/istio/tests/integration/helm/upgrade=*
+istio.io/istio/tests/integration/pilot/revisions=*
+istio.io/istio/tests/integration/security/sds_ingress/quic=*
```

Comment:
worth commenting why quic* test is skipped. (http3 ingress not avaialble in the testing env)